lib/grape/dsl/inside_route.rb

```diff
@@ -328,6 +328,8 @@ def sendfile(value = nil)
       # * https://github.com/rack/rack/blob/99293fa13d86cd48021630fcc4bd5acc9de5bdc3/lib/rack/chunked.rb
       # * https://github.com/rack/rack/blob/99293fa13d86cd48021630fcc4bd5acc9de5bdc3/lib/rack/etag.rb
       def stream(value = nil)
+        return if value.nil? && @stream.nil?
```

Comment:
This fixes a bug where headers re reset if you just do `.stream`. Write a spec for it please?